source/transfers/contracts/USDTTransferHandler.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity 0.5.12;
+
+import "./interfaces/ITransferHandler.sol";
+import "./interfaces/IUSDTTransferHandler.sol";
+
+contract USDTTransferHandler is ITransferHandler {
+
+  function transferTokens(
+    address _from,
+    address _to,
```

Comment:
underscores and misleading interface name

Reply:
removed this handler